packages/shared/type.ts

```diff
@@ -2,6 +2,7 @@ export interface ApiNote {
     id: string;
     content: string;
     summary: string;
+    style: Record<string, string>;
```

Comment:
Is `string` valid for all css properties or do you have to supply numbers for some properties?

Reply:
Yeah, react allows numbers but everything can be configured via strings.